css/screens/loginPage.css

```diff
@@ -0,0 +1,76 @@
+/* css for LogIn Page */
+
+
+header .left .brand-heading{
```

Comment:
don't set high specificity to any styles, unless necessart

Reply:
Actually i was using !important, then someone suggested me to add add a class.